components/brave_ads/browser/ads_service.h

```diff
@@ -18,6 +18,10 @@ class AdsService : public KeyedService {
   AdsService() = default;
 
   virtual bool is_enabled() const = 0;
+  virtual uint64_t ads_per_hour() const = 0;
```

Comment:
as mentioned above, adding getters/setters here isn't really necessary because we need to work with the prefs directly and that is the typical pattern in chromium so we can get notifications of changes through the PrefChangeRegistrar